traffic_ops/traffic_ops_golang/deliveryservice/deliveryservices.go

```diff
@@ -110,6 +110,42 @@ func (ds *TODeliveryService) IsTenantAuthorized(user *auth.CurrentUser) (bool, e
 	return isTenantAuthorized(ds.ReqInfo, &ds.DeliveryServiceV4)
 }
 
+const getTLSVersionsQuery = `
+SELECT tls_version
+FROM public.deliveryservice_tls_version
+WHERE deliveryservice = $1
+ORDER BY tls_version
+`
+
+// GetDSTLSVersions retrieves the TLS versions explicitly supported by a
+// Delivery Service identified by dsID. This will panic if handed a nil
```

Comment:
I don't know that's obvious without reading the function. It might be obvious to you, being familiar with how the TO codebase most often has functions that panic when given nil transactions, but that doesn't mean it's obvious to everyone. You could reasonably think that it would return an error instead of crashing the whole thread it's running on.

I don't think it hurts anything to make it clear in the GoDoc.

Reply:
> It might be obvious to you, being familiar with how the TO codebase most often has functions that panic when given nil transactions

The audience for these comments is TO developers, who will surely learn quickly that you can't get anywhere with a nil DB transaction when you need to access the DB. I don't care strongly enough to -1 this PR for these few instances, but I would -1 a PR that added these comments to every function of this kind.